packages/css/card.css

```diff
@@ -13,178 +13,176 @@
+    --dsc-card-backround: var(--ds-color-neutral-background-subtle);
+
+    &[data-link] {
+      --dsc-card-border-color: var(--ds-color-neutral-border-default);
+
+      &:hover {
+        --dsc-card-backround: var(--ds-color-neutral-surface-default);
+      }
+
+      &:active {
+        --dsc-card-backround: var(--ds-color-neutral-surface-hover);
+      }
+    }
+  }
+
+  &[data-color='brand1'] {
+    --dsc-card-border-color: var(--ds-color-brand1-border-subtle);
+    --dsc-card-backround: var(--ds-color-brand1-surface-default);
+    --dsc-card-color: var(--ds-color-brand1-text-default);
+
+    &[data-link] {
+      --dsc-card-border-color: var(--ds-color-brand1-border-default);
+
+      &:hover {
+        --dsc-card-backround: var(--ds-color-brand1-surface-hover);
+      }
+
+      &:active {
+        --dsc-card-backround: var(--ds-color-brand1-surface-active);
+      }
+    }
+  }
+
+  &[data-color='brand2'] {
+    --dsc-card-border-color: var(--ds-color-brand2-border-subtle);
+    --dsc-card-backround: var(--ds-color-brand2-surface-default);
+    --dsc-card-color: var(--ds-color-brand2-text-default);
+
+    &[data-link] {
+      --dsc-card-border-color: var(--ds-color-brand2-border-default);
+
+      &:hover {
+        --dsc-card-backround: var(--ds-color-brand2-surface-hover);
+      }
+
+      &:active {
+        --dsc-card-backround: var(--ds-color-brand2-surface-active);
+      }
+    }
+  }
+
+  &[data-color='brand3'] {
+    --dsc-card-border-color: var(--ds-color-brand3-border-subtle);
+    --dsc-card-backround: var(--ds-color-brand3-surface-default);
+    --dsc-card-color: var(--ds-color-brand3-text-default);
+
+    &[data-link] {
+      --dsc-card-border-color: var(--ds-color-brand3-border-default);
+
+      &:hover {
+        --dsc-card-backround: var(--ds-color-brand3-surface-hover);
+      }
+
+      &:active {
+        --dsc-card-backround: var(--ds-color-brand3-surface-active);
+      }
+    }
+  }
+
+  .ds-card__header h1,
+  .ds-card__header h2,
+  .ds-card__header h3,
+  .ds-card__header h4,
+  .ds-card__header h5,
+  .ds-card__header h6 {
+    color: inherit;
+  }
+
+  .ds-card__media > * {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    border: 0;
+  }
+
+  .ds-card__media {
+    width: auto;
+  }
+
+  .ds-card__media:first-child {
+    padding-bottom: var(--ds-spacing-4);
+  }
+
+  .ds-card__media:last-child {
+    padding-top: var(--ds-spacing-6);
+  }
+
+  .ds-card__footer,
+  .ds-card__content {
+    display: flex;
+    justify-content: flex-start;
+    gap: var(--ds-spacing-4);
+    flex-wrap: wrap;
+    word-wrap: break-word;
+    padding: var(--ds-spacing-2) 0;
+    color: inherit;
+  }
+
+  .ds-card__content {
+    flex-direction: column;
+  }
+
+  .ds-card__header {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    word-wrap: break-word;
+    font-family: inherit;
+    padding: var(--ds-spacing-2) 0;
+  }
+
+  & a,
+  & a:visited {
```

Comment:
Kunne vi brukt bare `a:any-link` i stede for 2 selectors? 😊

Reply:
Så lenge det ikkjer er eit problem at vår `Link` komponent bruker `:visted`, så skal det gå fint.
Bruker me to selectors er me sikre på at det ikkje blir kluss i fargen som blir satt.

Reply:
ja, `a:any-link` er like sterk som `a:visited` så det virker :)